circuits/voteverifier/inputs_test.go

```diff
@@ -0,0 +1,176 @@
+package voteverifier
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"os"
+
+	gecdsa "github.com/consensys/gnark-crypto/ecc/secp256k1/ecdsa"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/iden3/go-iden3-crypto/babyjub"
+	"github.com/iden3/go-iden3-crypto/poseidon"
+	"github.com/iden3/go-rapidsnark/prover"
+	"github.com/iden3/go-rapidsnark/witness"
+	"github.com/vocdoni/vocdoni-z-sandbox/ecc"
+	bjj "github.com/vocdoni/vocdoni-z-sandbox/ecc/bjj_gnark"
+	"github.com/vocdoni/vocdoni-z-sandbox/encrypt"
+	"go.vocdoni.io/dvote/util"
+)
+
+func GenerateECDSAaccount() (*ecdsa.PrivateKey, ecdsa.PublicKey, common.Address, error) {
```

Comment:
Document all these functions. Add the "Test" Prefix and make them private if possible.

Reply:
In fact i will need these functions for the following circuit so probably i will move them to a `test` package